pkg/util/kubernetes/hpa/datadog_external.go

```diff
@@ -0,0 +1,49 @@
+package hpa
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"gopkg.in/zorkian/go-datadog-api.v2"
+
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// QueryDatadogExternal converts the metric name and labels from the HPA format into a Datadog metric.
+// It returns the last value for a bucket of 5 minutes,
+func QueryDatadogExternal(metricName string, tags map[string]string) (int64, error) {
+	if metricName == "" || len(tags) == 0 {
+		return 0, errors.New("invalid metric to query")
+	}
+	bucketSize := config.Datadog.GetInt64("hpa_external_metric_bucket_size")
+	client := datadog.NewClient(config.Datadog.GetString("api_key"), config.Datadog.GetString("app_key"))
```

Comment:
do we want to create a client each time or could we keep the client?

Reply:
`HPAWatcherClient` could be passed an instance of `datadog.Client` in constructor. This would make it easier to mock and test.